tidy3d/components/lumped_element.py

```diff
@@ -146,32 +176,56 @@ def _voltage_axis_in_plane(cls, val, values):
             )
         return val
 
+
+class LumpedResistor(RectangularLumpedElement):
+    """Class representing a rectangular lumped resistor. Lumped resistors are appended to the list
+    of structures in the simulation as :class:`Medium2D` with the appropriate conductivity given
+    their size and voltage axis."""
+
+    resistance: pydantic.PositiveFloat = pydantic.Field(
+        ...,
+        title="Resistance",
+        description="Resistance value in ohms.",
+        unit=OHM,
+    )
+
     @cached_property
     def _sheet_conductance(self):
         """Effective sheet conductance."""
-        lateral_axis = 3 - self.voltage_axis - self.normal_axis
-        return self.size[self.voltage_axis] / self.size[lateral_axis] / self.resistance
+        size_voltage = self.size[self.voltage_axis]
+        size_lateral = self.size[self.lateral_axis]
+        return size_voltage / size_lateral / self.resistance
 
```

Comment:
Thanks @dmarek-flex ! One question regarding general lumped elements, do we treat lumped elements as 2D sheet rather than a 3D structure from Taflove? As the conductance here is defined by dz/dx/R rather than dz/(dx*dy)/R.

Reply:
Looking forward to the notebook so I can play with examples.

Reply:
This part is not very clear, but I based my new implementation on the old `LumpedResistor` implementation. We treat lumped elements as 2D sheets currently that end up becoming volumetric when the Medium2D is added to the simulation. The final scaling along the normal axis of the 2D sheet is applied there and the resulting thickness is equivalent to the grid spacing.